lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -338,14 +364,16 @@ def get_image_states_data(
         type,
         apply_filters=command_arg_filter
     )
-    return __process(url, info_type, command_arg_filter, result_format)
+    return __process(url, info_type, command_arg_filter,
+                     result_format, transfer_compression)
 
 
 def get_server_types_data(
         framework,
         type,
         result_format='plain',
         region='all',
+        transfer_compression='bz2',
```

Comment:
ditto, API breakage, and there is not a technical necessary reason to break the API.